Resources/Prototypes/Entities/Objects/Tools/welders.yml

```
@@ -199,6 +215,16 @@
       reagents:
       - ReagentId: WeldingFuel
         Quantity: 1
+  - type: Tool
+    speedModifier: 1.5 # For whatever reason this item didn't have this when an advanced industrial welding tool did
+    # Floof Shitmed - Improvised Tools
+  - type: Cautery
+    speed: 1.3
```

Refer to the comment on the `speedModifier`.
```suggestion
    speed: 1.0
```